test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs

```diff
@@ -62,6 +63,12 @@ public void AssertInstalledPackageNotDeprecated()
             DeprecatedPackageResult.Should().BeFalse();
         }
 
+        public void AssertPackageNameAndType(string packageId, PackageLevel packageLevel)
+        {
+            var package = _uiproject.GetPackageItemOnInstalledTabById(packageId);
+            package.PackageLevel.Should().Be(packageLevel);
```

Comment:
If the above method returns null, this would throw a null ref.

Reply:
Hi @nkolev92 , thanks for your review and suggestion, we added two helper validations for null refs to this Assert method, the first to avoid an empty package list on the "Installed" tab, and the second to avoid a package list that doesn't contain the currently validated package. Please help to review them again. Thanks a lot.